src/app/globals.css

```diff
@@ -1,18 +1,13 @@
 @font-face {
   font-family: 'Lato';
-  src: url('assets/fonts/Lato-Light.ttf');
-  font-weight: 300;
+  src: url('./assets/fonts/Lato-Light.ttf');
+  src: url('./assets/fonts/Lato-Regular.ttf');
+  font-weight: 300, 400;
```

Comment:
Does this work? I remember trying it like this a while ago, and it wouldn't. Looks good though, if it works

Reply:
It does not work. It was my suggestion to do it this way because I have seen it done by others. I just tested it and I had to separate the font-weights.

Reply:
for me it seems to work